packages/rhsplib/CMakeLists.txt

```diff
@@ -3,7 +3,8 @@ cmake_policy(SET CMP0091 NEW)
 cmake_policy(SET CMP0042 NEW)
 set (CMAKE_CXX_STANDARD 17)
 
-add_subdirectory(librhsp)
+# By specifying EXCLUDE_FROM_ALL, we prevent the tests from being built
```

Comment:
EXCLUDE_FROM_ALL also excludes building librhsp. Can we just exclude the tests somehow? Possibly through a define.